CHANGELOG-2.0.md

```diff
@@ -0,0 +1,19 @@
+# Changelog since v1.2.0
+
+The version 2.0 is not compatible with v1.x. See Action Required section and update CSI driver manifests.
+
+## Action Required
+
+- The external-attacher now uses PATCH HTTP method to update API objects. Please update the attacher RBAC policy to allow the attacher to `patch`  VolumeAttachments and PersistentVolumes. See `deploy/kubernetes/rbac.yaml` for an example. ([#177](https://github.com/kubernetes-csi/external-attacher/pull/177), [@jsafrane](https://github.com/jsafrane))
+- The `-connection-timeout`, `-leader-election-type` and `-leader-election-identity` flags, deprecated in v1.2, have been removed. Please update your manifests for the external-attacher.
```

Comment:
Also mention here that only the lease type is supported, and leader election will not properly work when doing a rolling upgrade from a version that previously used the configmaps type.

Reply:
Fixed